motion/p5js/motion-triangle/index.js

```diff
@@ -0,0 +1,75 @@
+function Color(color){
```

Comment:
Is this really necessary? you can just use the values. Seems a bit overcomplicated. Also P5.js has already a color function. See --> http://p5js.org/reference/#/p5/color